mkdocs.yml

```diff
@@ -68,6 +68,7 @@ nav:
         - modules/pulsar.md
         - modules/redis.md
         - modules/redpanda.md
+        - modules/dockerregistry.md
```

Comment:
This entry will be probably located at L63 https://github.com/testcontainers/testcontainers-go/pull/1165/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R63